src/Controller/Admin/DashboardController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use EasyCorp\Bundle\EasyAdminBundle\Config\Dashboard;
+use EasyCorp\Bundle\EasyAdminBundle\Config\MenuItem;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractDashboardController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class DashboardController extends AbstractDashboardController
+{
+    /**
+     * @Route("/admin", name="admin")
+     */
+    public function index(): Response
+    {
+        return parent::index();
+    }
+
+    public function configureDashboard(): Dashboard
+    {
+        return Dashboard::new()
+            ->setTitle('App');
```

Comment:
We should add a `SITE_TITLE` environment variable (or similar) for this value and expose it in a service parameter. Then we can use `$this->getParameter` to get it here and set the title.

Reply:
Done in a6b7ba0